packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -335,6 +421,49 @@ class _BottomSheetState extends State<BottomSheet> {
 
 typedef _SizeChangeCallback<Size> = void Function(Size);
 
+class _DragHandle extends StatelessWidget {
+  const _DragHandle({
+    required this.onSemanticsTap,
+    required this.handleHover,
+    required this.materialState,
+  });
+
+  final VoidCallback? onSemanticsTap;
+  final Function(bool) handleHover;
+  final Set<MaterialState> materialState;
+
+  @override
+  Widget build(BuildContext context) {
+    final BottomSheetThemeData bottomSheetTheme = Theme.of(context).bottomSheetTheme;
+    final BottomSheetThemeData m3Defaults = _BottomSheetDefaultsM3(context);
+    final Size dragHandleSize = bottomSheetTheme.dragHandleSize ?? m3Defaults.dragHandleSize!;
```

Comment:
Need to include BottomSheet.dragHandleSize here. Probably want to pass it in as a _DragHandleParameter.